### PR TITLE
Add radius and textVisible props (closes Issue #1)

### PR DESCRIPTION
**Pull request by pkrawc**
PR for Issue #1

### CODE REVIEW

File: src/ProgressArc.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ function ProgressArc({
       />
 
       <text x="100" y="115">
-        {value}{unit}
+        { textVisible ? {`${value} ${unit}`} : null }
```
Comment (szastupov) [inline src/ProgressArc.js]: Hmm, maybe in this case we shouldn't render text node at all?

**Comment by szastupov** [comment]
Also, could you update README.md as well?

Thanks for the PR 🙏

File: src/ProgressArc.js (hunk 1 above)
Comment (pkrawc) [reply src/ProgressArc.js]: huh, yea. I don't really know why I didn't write it that way from the get-go.

**Comment by pkrawc** [comment]
Changes made. I'll let you close the PR.

File: src/ProgressArc.js [hunk 2]
```diff
@@ -31,13 +31,11 @@ const Svg = styled.svg`
   }
 `
 
-const radius = 90
-const p = 2 * radius * Math.PI
-
 function ProgressArc({
   value, max, className, unit,
-  arcColor, arcBackgroundColor, textColor, rounded
+  arcColor, arcBackgroundColor, textColor, radius, rounded
```
Comment (szastupov) [inline src/ProgressArc.js]: You forgot to add textVisible here.

File: README.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ import ProgressArc from 'progress-arc-component'
 - `arcColor` - Progress arc color (#818a91)
 - `arcBackgroundColor` - Arc background color (#eceeef)
 - `textColor` - Text color (#818a91)
+- `radius` - Set the radius of the arc (90)
```
Comment (szastupov) [inline README.md:26]: and  add textVisible here.

**Review by szastupov — CHANGES_REQUESTED** [review]

File: src/ProgressArc.js (hunk 2 above)
Comment (pkrawc) [reply src/ProgressArc.js]: Thats what you get for writing without a dev environment going at 11pm

File: README.md (hunk 3 above)
Comment (pkrawc) [reply README.md:26]: Ditto for this

**Comment by szastupov** [comment]
Great, thanks!

**Pull request merged**